src/pages/profile-edit.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Profile Edit</title>
+ 
+  </head>
+  <body>
+    {{> header}}
+    <div class="flex justify-center bg-[#F5F5F5]">
+   <div class="m-10 flex flex-col gap-y-6 rounded  p-3 " dir="rtl">
+    <form class="flex flex-col gap-3  bg-[#FFFFFF] h-[528px] w-[986px] ">
+      <h2 class="w-[249px] h-[36px]  not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
```

Comment:
Remove all fixed widths & heights. Elements should have dynamic size. Instead, use `line-height` to set the element's inner height.

```suggestion
      <h2 class="not-italic font-bold text-[20px] leading-[36px] m-5">ویرایش اطلاعات شخصی</h2>
```